packager/packager.js

```diff
@@ -169,7 +173,7 @@ function getDevToolsLauncher(options) {
       res.writeHead(200, {'Content-Type': 'text/html'});
       fs.createReadStream(debuggerPath).pipe(res);
     } else if (req.url === '/launch-chrome-devtools') {
-      var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+      var debuggerURL = options.origin + ':' + options.port + '/debugger-ui';
```

Comment:
Why do you need this? The URL is served on the same host the server is running. Or am I missing something?